controllers/activemqartemis_controller_test.go

```diff
@@ -3888,7 +3887,7 @@ var _ = Describe("artemis controller", func() {
 			Expect(createdCrd.Name).Should(Equal(crd.ObjectMeta.Name))
 
 			By("By finding a new config map with broker props")
-			configMap := &corev1.ConfigMap{}
+			configMap := &corev1.Secret{}
```

Comment:
could you rename the configMap variables to align them with theirs new type?

Reply:
yep, i'll have that in a later PR, I left them intentionally unchanged to make the diff visible